manifests/base/files/gen-istio-cluster.yaml

```diff
@@ -13,7 +13,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: istio-pilot-service-account
-  namespace: istio-system
+  namespace: default
```

Comment:
This is going to need to be in istio-system. may need to modify the `gen-kustomize` target to pass the namespace?